fast_response/FastResponseAnalysis.py

```diff
@@ -992,6 +1020,424 @@ def unblind_TS(self, custom_events=None):
+            xsize=nPixZoom,ysize=nPixZoom,reso=reso,bgcolor="white",badcolor="white",margins=(.02,.02,.05,0),notext=True,cbar=False)
+            
+        hp.graticule(verbose=True)
+        plotting_utils.plot_labels(self.skymap_fit_dec,self.skymap_fit_ra, reso,nPix=nPixZoom,label_scale=.97)
+        
+        ax=plt.gca()
+        text_objs = ax.findobj(
+                lambda artist: isinstance(artist, Text)
+            )
+        for obj in text_objs:
+            obj.set_fontsize(30)
+        plotting_utils.plot_color_bar(range=[0,np.nanmax(self.skymap)], cmap="viridis", col_label=r"Skymap Prior PDF",
+                    offset=-50,labels=[0, "{0:.1e}".format(np.nanmax(self.skymap)/2),"{0:.2e}".format(np.nanmax(self.skymap))],loc=[0.86, 0.2, 0.03, 0.6])
+        theta, phi =plotting_utils.plot_contours([.9],self.skymap)
+        hp.projplot(theta[0], phi[0], linewidth=2., c='k')
+        for i in range(1, len(theta)):
+            hp.projplot(theta[i], phi[i], linewidth=2., c='k', label=None)
+        name="PriorSkymap"
+        if(not format):
+            name+="Unformatted"
+        plt.savefig(plotting_location+name+".png")
+
+    def _make_posterior_fits(self):
+        #internal function to be called from make_posterior_skymap
+        #saves the skymaps as a fits file
+        extra_header = [('index', self.index),
+                   #('energy_range', ),
+                   #('ns_range', ),
+                   ("Sender","IceCube Collaboration"),
+                   ("TRIGGER_EVENT",self.name),
+                   ("startmjd",self.start),
+                   ("stopmjd",self.stop)
+
+                   ]
+        #startmjd, stopmjd, 
+        hp.write_map(self.analysispath + '/' +'posterior_info.fits',m=self.PosteriorSkymap,
+                    extra_header=extra_header,partial=True    
+                              
+                              )
+   
+        
+    def fluxToAverageNs(self,flux):
+        if(self.inj==None):
+            print("Initialize the injector. Can't convert flux to ns without an injector")
+        return self.inj.flux2mu(flux)    
+    def fluxToNs_decDependent(self,flux,dec):
+        mean_signal=self.fluxToAverageNs(flux)
+        acceptance_weights = self.inj.relative_signal_acceptance(np.sin(dec))
+        num=[np.array(np.around(w * mean_signal / float(self.prior.nprior)),
+                            dtype=np.int).tolist()
+                   for w in acceptance_weights]
+        num_sum = np.sum(num, dtype=int)
+        return num_sum
+    
+    def make_posterior_map(self, fluxToTest=[1,2,3,4,5], 
+                           prior_func=lambda ns,gamma,ra,dec: 1, 
+                           plotting_location=None,
+                           custom_events=None,
+                           useFlux=False
+                           ):
+        # Posterior Approach
```

Comment:
the info here for the docstring is great! can you reformat so that the docs includes all of it properly?